manual_user_install.sh

```diff
@@ -0,0 +1,238 @@
+    # we install full tensorflow (because it just works, takes a while to install)
+    # sudo apt install -y libatlas-base-dev
+    #  pip3 install tensorflow --no-cache-dir
+    pip3 install tflite_runtime
+    pip3 install PyYAML
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-workshop
+    sudo apt install -y python3-fann2
+    pip3 install padatious
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-lingua-franca
+    pip3 install PyAudio
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-pocketsphinx
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-precise
+    # pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-vosk
+
+    # install speech to text
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-stt-plugin-server
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-stt-plugin-selene
+    # pip3 install git+https://github.com/OpenVoiceOS/ovos-stt-plugin-vosk
+
+    # install text to speech
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-microphone-plugin-alsa
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic3-server
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-server-plugin
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic2
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-google-tx
+
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-config
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-utils
+    sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/fingerprinting.py
+    sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/configuration.py
+    sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/ovos_service_api.py
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-bus-client
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-plugin-manager
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-workshop
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-lingua-franca
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-cli-client
+
+    # install phal components
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-PHAL
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-connectivity-events
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-system
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-PHAL-plugin-ipgeo
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-PHAL-plugin-oauth
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-dashboard
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-alsa
+
+    sudo apt install -y pulseaudio
+    pulseaudio -D
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-pulseaudio
```

Comment:
do not install this, its still in testing and replaces alsa, we have some bug reports and recommend staying with alsa plugin installed above, you are supposed to use only one or the other (pulseaudio is given preference)

Reply:
Yes, `ovos-phal-plugin-alsa` works much more reliably for me

Reply:
Thank you! Pulse is a bit of a pain. Gone in my copy. Will test.

Reply:
works

Reply:
I spoke too soon.  This morning I need
    sudo apt install -y pulseaudio
    pulseaudio -D
or I get
ovos-systemd-phal[6080]: pa_context_connect() failed: Connection refused
ovos-systemd-dinkum-listener[6134]: pa_context_connect() failed: Connection refused
Let me resolve this one last.

Reply:
Yes, pulseaudio needs installed, but the plugin does not

Reply:
My understanding now is that pulse needs to be installed but not the plugin.  All good.